### DISCUSSION TITLE
Ayuda con mi equipo

### DISCUSSION THREAD

**Discussion by ErickBarreraO** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#557] [Q&A] [answered]
Hola compañeros, buenas tardes, disculpen que mi consulta no sea referente a la misión, espero no haya ningún problema. Ando un poco atrasado con mis prácticas ya que no he podido iniciar sesión en windows en mi equipo, me dice que hay un error al iniciar sesión, ya acudí a las respuestas que da microsoft y he podido crear un nuevo usuario con permisos de administrador pero el problema es que no me deja accesar a mis archivos que tenía en la otra cuenta (incluidos los de launch X), no se si alguien sabe como puedo recuperarlos. 
Gracias.

**Comment by ismagician** [comment]
¿ Podrías mostrar lo que te sale cuando quieres accesar a los archivos de tu usuario anterior?
Por lo general si eres admin y quieres accesar a C:\Users\TuUsuario te sale un mensaje,  le das continuar y te deja 
![imagen](https://user-images.githubusercontent.com/92187317/160721401-531d8aed-40c1-4f9c-8709-002e36af3804.png)

**Reply by ErickBarreraO** [reply]
Si, me sale esa ventana pero al darle clic se queda cargando, cuando termina no abre la carpeta y al hacer clic de nuevo me vuelve a preguntar, no se si habrá alguna otra forma de acceder a ellos

**Comment by Jimdrer** [comment]
Específica que tipo de error, tal vez se pueda dar solución.

**Reply by ErickBarreraO** [reply]
Cuando inicio sesión me sale el mensaje "No podemos iniciar sesión en tu cuenta" y me dice que intenté cerrando sesión y volviendo a intentar (ya lo hice y no funcionó, también la reinicié y pasó lo mismo), luego sale una notificación que menciona que me ha dado acceso al equipo con una cuenta temporal y no puedo abrir ninguno de los archivos que tengo en la otra cuenta. porque esa es la de administrador. Investigando un poco en las respuestas de microsoft menciona que puede deberse a algún efecto del antivirus o el propio microsoft protect, ya los deshabilité y siguió, entonces parece ser que se debe a algún daño en el perfil de usuario.

**Comment by doguedogue** [comment, marked as answer]
Entra en modo a prueba de fallos y cambiar los permisos
